src/Components/BookForm/BookForm.jsx

```diff
@@ -22,6 +23,12 @@ const BookForm = ({ authors, handleFormSubmit }) => {
           type="text"
           placeholder="Book title"
         />
+        <input
+          onInput={(ev) => setBook((prev) => ({ ...prev, category: ev.target.value }))}
+          name="book-category"
+          type="text"
+          placeholder="Book category"
+        />
```

Comment:
**OPTIONAL** - To help improve the UX of your app, I suggest making it impossible to add a book without an author or book category. Please fix this 🙏 

Your Project:
- I was able to add a book without an author and book category 
<img width="823" alt="3vrvrvv" src="https://user-images.githubusercontent.com/73607512/207612782-9a038f85-7000-498a-ba49-921c31c785cb.PNG">